packages-common/pluggable-widgets-tools/package.json

```diff
@@ -115,7 +115,10 @@
   },
   "scripts": {
     "prepublish": "rimraf dist && tsc",
-    "test": "cross-env LIMIT_TESTS=true npm run test:cross-platform",
-    "test:cross-platform": "jest --testTimeout=720000 tests/commands.spec.js"
+    "test": "npm run test:src:web && npm run test:src:native",
```

Comment:
The idea was to always run a small subset of scripts tests during `npm test`. Otherwise `LIMIT_TESTS` option really looses its purpose. So I would say either bring it back (you might need to limit tests more agressively than before) or remove `LIMIT_TESTS` option